tests/jballerina-unit-test/src/test/resources/test-src/expressions/let/let-expression-test.bal

```diff
@@ -316,6 +316,18 @@ function getRecordConstrainedError() returns FooError {
     return e;
 }
 
+public function testAnonymousRecordWithLetExpression() {
+    record {
+        int i;
+        int j?;
+    } rec = let int v = 1 in {i: v};
+    
+    assert(rec.i, 1);
```

Comment:
The expected value seems to be the first param.
```suggestion
    assert(1, rec.i);
```

Reply:
Done!